hw/top_earlgrey/data/chip_testplan.hjson

```diff
@@ -13,6 +13,160 @@
                      "hw/ip/tlul/data/tlul_testplan.hjson"]
 
   testpoints: [
+
+    //////////////////
+    // Host Device  //
+    // RV_CORE_IBEX //
+    //////////////////
+    //
+    // Note: the Debug Module will be referred to as RVDM.
+    {
+      name: chip_ibex_debug_basic
+      desc: '''Verify correct operation of rv_core_ibex while in debug mode.
+
+            Trigger the RVDM to send a debug request to rv_core_ibex, which should then jump to the
+            debug ROM and execute the code there before returning via `dret` and executing the rest
+            of the main program.
+            '''
+      milestone: V2
+      tests: []
+    }
+    {
+      name: chip_ibex_debug_single_step_test
```

Comment:
Part of this should be covered by covered by [rom_e2e_debug](https://github.com/lowRISC/opentitan/blob/a00ce54bf6341a296c0d3864008487e0b65e5787/sw/device/silicon_creator/rom/data/rom_testplan.hjson#L787) and [rom_e2e_jtag_inject](https://github.com/lowRISC/opentitan/blob/a00ce54bf6341a296c0d3864008487e0b65e5787/sw/device/silicon_creator/rom/data/rom_testplan.hjson#L816), although these tests do not perform multiple single steps.

Should we extend [rom_e2e_debug](https://github.com/lowRISC/opentitan/blob/a00ce54bf6341a296c0d3864008487e0b65e5787/sw/device/silicon_creator/rom/data/rom_testplan.hjson#L787) to perform more steps?

Reply:
@andreaskurth can you take this one into account when updating existing tetspoints such as [chip_rv_dm_perform_debug](https://www.google.com/url?q=https://github.com/lowRISC/opentitan/blob/6f7094c7b1fc37b0a1955a04d3c259aab86d50c5/hw/top_earlgrey/data/chip_testplan.hjson%23L548&sa=D&source=docs&ust=1663272181668386&usg=AOvVaw3tIZfeqvAfzelQUevBPR1_).